dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -331,34 +331,34 @@ BEGIN
     EDITTEXT 14008, 197, 103, 33, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     CONTROL "", 14015, "Static", SS_OWNERDRAW, 70, 116, 100, 40
     CTEXT "Partiție %c", 14009, 95, 162, 50, 10
-    PUSHBUTTON "Curățare &disc…", 14010, 160, 159, 70, 15, WS_TABSTOP
+    PUSHBUTTON "Curăţare &disc", 14010, 160, 159, 70, 15, WS_TABSTOP
     LTEXT "", -1, 8, 176, 223, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
-    CHECKBOX "&Comprimă volumul pentru a economisi spațiu", 14011, 8, 190, 222, 10, WS_DISABLED
-    CHECKBOX "Permite „Serviciu de inde&xare” pentru accelerarea căutărilor", 14012, 8, 204, 222, 10, WS_DISABLED
+    CHECKBOX "&Comprimare unitate pentru economie de spaţiu-disc", 14011, 8, 190, 222, 10, WS_DISABLED
+    CHECKBOX "Serviciul de Indexare va &indexa acest disc pentru căutare rapidă", 14012, 8, 204, 222, 10, WS_DISABLED
```

Comment:
Are you really sure that MS assigned an i as accelerator? That is something that MS usually does not recommend. Screenshot therefore of the MS shell32 which does prove that accel.

Reply:
![2024-03-27 00_49_16-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/9f6ea923-b9db-4d43-af9f-82e92eef1079)

Reply:
Okay, resolved then. MS is king.